pkg/syncer/spec/resources.go

```diff
@@ -0,0 +1,126 @@
+/*
+Copyright 2022 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package spec
```

Comment:
Do you think we could move the dns-related files (the yaml files, resources.go), and also possibly the dns-related `processXXX` method definitions (gathered in a struct ?) into a `dns` package ?

Reply:
done.